<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoContact</class>
 <widget class="QWidget" name="InfoContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>409</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelPhoto">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Last name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Entreprise</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Mail</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineFirstName"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineLastName"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEntreprise"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineMail"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="linePhone"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="interactionLayout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
